engine/engine/content/builtins/manifests/android/AndroidManifest.xml

```diff
@@ -13,6 +13,7 @@
         {{#has-icons?}}
         android:icon="@drawable/icon"
         {{/has-icons?}}
+        android:extractNativeLibs="{{android.extract_native_libs}}"
```

Comment:
For all the changes in `AndroidManifest.xml` it's useful to add a warning that now users have to update their custom Android Manifests.